znoyder/browser.py

```diff
@@ -70,6 +71,17 @@ def get_packages(**kwargs):
     return packages
 
 
+def get_projects_mapping(**kwawrgs) -> list:
+    packages = get_packages()
```

Comment:
`packages = get_packages()` -> `packages = get_packages(**kwawrgs)` would allow the filtering using the same parameters as for `get_packages()`, which IMO may be useful in the future